paranoia.gemspec

```
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.5'
 
-  s.add_dependency 'activerecord', '>= 5.1', '< 7.1'
+  s.add_dependency 'activerecord', '>= 5.1', '<= 7.1'
```

I think that this should be

```suggestion
  s.add_dependency 'activerecord', '>= 5.1', '< 7.2'
```

so that patch versions of 7.1 are also permitted.